src/views/Notifications/Home/_components/_utils.tsx

```diff
@@ -104,39 +104,42 @@ export const ActivityCTA = ({
   return <></>;
 };
 
-export const pushValuesToChangesArray = (
-  obj: ChangeTypes,
-  changesArray: string[]
-) => {
-  Object.entries(obj).forEach(([key, value]) => {
-    if (key !== '__typename') {
-      // Ignore key values that where the key is '__typename'
-      if (Array.isArray(value) && value.length > 0) {
-        // Ignore empty arrays (i.e. [])
-        if (key === 'added') {
-          // Model Lead added is important, so pushed to the front of array to ensure it will always be shown
-          changesArray.unshift(key);
-        } else {
+export const pushValuesToChangesArray = (obj: ChangeTypes): string[] => {
```

Comment:
Thanks for the comments, I think it would be useful to have an understanding of what this function does as at a high level, before the function is defined.  The name of the function is a bit generic, so a comment to describe what it's doing as a whole. Maybe mention where the data is coming from and why it needs to be transformed in this way. Also not sure what the keys being pushed to the array represent

After logging the output, it seems like the return type should be a certain set, so can probably replace 
`(obj: ChangeTypes): string[]`
with
`(obj: ChangeTypes): 'added' | 'count' | 'updated'` etc
This will make it more strongly typed and also inform about the shape/content of the return